src/sentry/static/sentry/app/views/settings/organizationDeveloperSettings/sentryApplicationDashboard/requestLog.tsx

```diff
@@ -0,0 +1,329 @@
+                {t('Errors Only')}
+              </ErrorsOnlyCheckbox>
+            </Button>
+          </RequestLogFilters>
+        </div>
+
+        <Panel>
+          <PanelHeader>
+            <TableLayout>
+              <div>{t('Time')}</div>
+              <div>{t('Status Code')}</div>
+              {app.status !== 'internal' && <div>{t('Organization')}</div>}
+              <div>{t('Event Type')}</div>
+              <div>{t('Webhook URL')}</div>
+            </TableLayout>
+          </PanelHeader>
+
+          {!this.state.loading ? (
+            <PanelBody>
+              {currentRequests.length > 0 ? (
+                currentRequests.map((request, idx) => (
+                  <PanelItem key={idx}>
+                    <TableLayout>
+                      <TimestampLink date={request.date} />
+                      <ResponseCode code={request.responseCode} />
+                      {app.status !== 'internal' && request.organization && (
+                        <div>{request.organization.name}</div>
+                      )}
+                      <div>{request.eventType}</div>
+                      <OverflowBox>{request.webhookUrl}</OverflowBox>
+                    </TableLayout>
+                  </PanelItem>
+                ))
+              ) : (
+                <EmptyMessage icon="icon-circle-exclamation">
+                  {t('No requests found in the last 30 days.')}
+                </EmptyMessage>
+              )}
+            </PanelBody>
+          ) : (
+            <LoadingIndicator />
+          )}
+        </Panel>
+
+        <PaginationButtons>
+          <Button
+            icon="icon-chevron-left"
+            onClick={this.handlePrevPage}
+            disabled={!this.hasPrevPage}
+            label={t('Previous page')}
+          />
+          <Button
+            icon="icon-chevron-right"
+            onClick={this.handleNextPage}
+            disabled={!this.hasNextPage}
+            label={t('Next page')}
+          />
+        </PaginationButtons>
+      </React.Fragment>
+    );
+  }
+}
+
+const TableLayout = styled('div')`
+  display: grid;
+  grid-template-columns: 1fr 0.5fr 1fr 1fr 1fr;
+  grid-column-gap: ${space(1.5)};
+  width: 100%;
+  align-items: center;
+`;
+
+const OverflowBox = styled('div')`
+  word-break: break-word;
+`;
+
+const PaginationButtons = styled('div')`
+  display: flex;
+  justify-content: flex-end;
+  align-items: center;
+
+  > :first-child {
+    border-top-right-radius: 0;
+    border-bottom-right-radius: 0;
+  }
+
+  > :nth-child(2) {
+    margin-left: -1px;
+    border-top-left-radius: 0;
+    border-bottom-left-radius: 0;
+  }
+`;
+
+const RequestLogFilters = styled('div')`
+  display: flex;
+  align-items: center;
+  padding-bottom: ${space(1)};
+
+  > * + * {
+    margin-left: ${space(1)};
+  }
+`;
+
+const ErrorsOnlyCheckbox = styled('div')`
+  display: flex;
+  align-items: center;
+  justify-content: flex-end;
+
+  > input {
```

Comment:
Can we use `styled` to style the `input` directly instead of using descendent selector? Alternatively, we can use `grid` and define a `grid-gap`.

Reply:
there's some weird stuff going on with some global css somewhere so the checkbox can't be styled directly (the global css is overriding it...)